best-practices/development-tools/azure/devops/continuous-integration/azure-app-functions-continuous-integration.md

```diff
@@ -0,0 +1,11 @@
+# Continuous Integration using Azure DevOps
+
+## .NET Core Unit Test
+1. Go to the Pipelines page in Azure DevOps and choose New Pipeline.
+2. Choose `Github Yaml`.
+3. Select GitHub as a source. (You might need to authorize Azure Pipelines with GitHub via OAuth. If so, choose "Authorize using OAuth ". This will open a new window to either log in with GitHub or grant permission to authorize Azure Pipelines with the logged in GitHub account. Granting access to the team organization where the repository is located might require an additional step: approval by an owner from that organization.) Select the target Repository and the default github branch then continue.
+4. On the Choose a Template page, select `ASP.NET Core` template and choose `Apply`.
+5. Get the pipeline from `azure-pipelines-azure-app-functions-ci.yml`.
```

Comment:
We may want to specify which kind of CI we are using at this step (CircleCI, GithubCI, etc...) just so we dont leave this up for interpretation.

Reply:
Actually the title for this paragraph is saying that `Continuous Integration using Azure DevOps`.

Reply:
We are talking about Azure DevOps CI.